requirements.txt

```diff
@@ -10,7 +10,7 @@ Flask-SQLAlchemy==2.3.2
 gunicorn==19.7.1
 isort==4.3.4
 itsdangerous==0.24
-Jinja2==2.10
+Jinja2==2.10.*
```

Comment:
Was there a point to adding the `.*`?

Reply:
It's been such a long while since I wrote that I can't remember. May have been a dependency vulnerability I just wanted to shut up.
I should go back (maybe at a later time) and update the version matching to `~=` instead of `==`